app/templates/__base_ui__.html

```diff
@@ -34,6 +35,23 @@ <h1 class="e-logo">{{ gettext('Network of Innovators') }}</h1>
     </nav>
 </header>
 
+<div class="modal" id="post-a-message" tabindex="-1" role="dialog">
+<div class="modal-dialog">
+<div class="b-post-update-wrapper m-update-active">
+<div class="e-post-update">
```

Comment:
Because we're shoehorning this into Bootstrap's modal plugin, it's not appearing as an extension of the top nav bar, as was originally envisioned. Instead, it's appearing on top of the nav bar... We can file a separate issue to fix this later, as there are multiple ways to do it and none of them seem particularly trivial to me.